### DISCUSSION TITLE
[Bug] Saving general settings

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#310] [Help] [answered]
Ahoi!

I opened a new account and tried to change the general settings (test environment). Also by changing minimal, i.e. setting timezone i receive the follwing error in backend:

Field error: Immutable field
```
{
  "field": "environments.0.parent_id"
}
```

**Comment by logeshswell** [comment]
Thanks for bringing it up. I can confirm that it is a bug. 
I will file a bug ticket and keep you posted if it is resolved.

**Reply by aanomm** [reply]
following for an update (as I have the same issue) 🤗

**Comment by ericingram** [comment]
Thanks for reporting. Increasing priority on this issue

**Comment by steven-k1** [comment, marked as answer]
@josias-hosang @fucha-co A fix was applied for this issue, and settings are editable within the test environment once again.
